submissions/Enochomosebi/htmlforms/pagenav.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
```

Comment:
You are not asked to create any new page. Kindly remove this file

Reply:
file has been removed